core/mondoo-linux-security.mql.yaml

```diff
@@ -1929,7 +1929,7 @@ queries:
         Edit or create a file in the `/etc/audit/rules.d/` directory ending in `.rules` and add the following line:
 
         ```
-        -w <Path to sudo logfile> -p wa -k actions
+        -w <Path to sudo log file> -p wa -k actions
```

Comment:
This looks like this is two args

Reply:
Everything in < > is a single argument. I think if folks are confused here they were probably confused before